src/app/modules/core/manager/landlord/components/edit/manager-landlord-edit.component.html

```diff
@@ -0,0 +1 @@
+<p>edit works!</p>
```

Comment:
Please, rename the file, the component class, all the related files and the selector in `landlord-edit` instead of `manager-landlord-edit`. Most probably this component will be shared with the landlord module. Thanks.